Source/Core/Plugins/EventParser/Raygun/RaygunEventParserPlugin.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using Exceptionless.Core.Extensions;
+using Exceptionless.Core.Models;
+using Exceptionless.Core.Pipeline;
+
+namespace Exceptionless.Core.Plugins.EventParser.Raygun {
+    [Priority(0)]
+    public class RaygunEventParserPlugin : IEventParserPlugin {
+        private readonly RaygunEventMapper _mapper = new RaygunEventMapper();
+
+        public List<PersistentEvent> ParseEvents(string input, int apiVersion, string userAgent) {
+            if (userAgent != "raygun" && apiVersion != 1)
```

Comment:
I guess we we could just do user agent != raygun  return, there api version may increase and we only want to support what we know.